### ISSUE TITLE
Don't scan user specified characters/symbols (Android)

### ISSUE THREAD

**Issue by shiki-tm** [themoeway/yomitan#1449]
I notice on my phone on some websites when I touch some text or miss the text as is pretty common on touch screens sometimes, some random stuff like punctuation or even unseeable text elements(?) trigger a popup. For example: 
![Screenshot_20240925_182526_Kiwi Browser](https://github.com/user-attachments/assets/624429a2-bdc8-4a68-b583-8d30771604ac)
![Screenshot_20240925_182705_Kiwi Browser](https://github.com/user-attachments/assets/f70f8c96-a869-4f1d-8797-221ae946c112)

Instead of adding them to regex and replacing with a whitespace which just causes other issues like connecting words and characters to the next sentence, can there be a feature where the user just adds to a list of things they don't want to ever be scanned and trigger a popup? 

There's clearly some things that don't ever need to be scanned but this depends on the user.

**Comment by Kuuuube** [comment]
Since you appear to be using Yomitan for a language with kanji. You can disable `Search text with non-Japanese, Chinese, or Cantonese characters` and it should resolve this issue.

**Comment by shiki-tm** [comment]
After turning off the setting and refreshing tabs it is still possible for some symbols to show up

![Screenshot_20241001_092258_Kiwi Browser](https://github.com/user-attachments/assets/b58ca383-9729-465f-9758-8c8a9854ea72)

**Comment by tadorituki** [comment]
hi does this issue still occour when using a different kanji dictionary, such as KANJIDIC
i suspect this might just be wiktionary漢字 having incorrectly added entries

**Comment by tadorituki** [comment]
maybe another option for scanning non-japanese text? something like
- scan all characters
- scan japanese characters (including punctuation and japanese special characters)
- only kana and kanji

**Comment by shiki-tm** [comment]
Yeah the thing with the brackets only happens with Wiktionary. 「」only these 2 have some kind of entry. 

Still I think if we could have a blacklist of stuff we don't want searched or causing popups it would be great especially because on some websites certain things are annoying like on this symbol or this phrase. Of course this is mostly an issue on a touch screen. 
![Screenshot_20241015_201007_Kiwi Browser](https://github.com/user-attachments/assets/97b497ce-7e8f-4ff4-90ed-7a305ba632f8)
![Screenshot_20241015_202509_Kiwi Browser](https://github.com/user-attachments/assets/aa5ef8e2-105d-4885-a732-6799ab2f80e7)
